tiles/src/main/java/com/protomaps/basemap/layers/Roads.java

```diff
@@ -134,23 +133,20 @@ public void processOsm(SourceFeature sf, FeatureCollector features) {
 
       var feat = features.line("roads")
         .setId(FeatureId.create(sf))
-        // Core Tilezen schema properties
         .setAttr("kind", kind)
         // To power better client label collisions
         .setAttr("min_zoom", minZoom + 1)
         .setAttrWithMinzoom("ref", shield.text(), minZoomShieldText)
         .setAttrWithMinzoom("shield_text_length", shieldTextLength, minZoomShieldText)
         .setAttrWithMinzoom("network", shield.network(), minZoomShieldText)
-        // Core OSM tags for different kinds of places
-        .setAttrWithMinzoom("layer", Parse.parseIntOrNull(sf.getString("layer")), 12)
```

Comment:
Tilezen baked the layer into sort_rank calculation... but we're using 400 everywhere in this PR so there would be information loss. Without being more sophisticated about the sort_rank, the more interesting highway junctions won't render correctly anymore.

Reply:
This first PR is just go get the key in there so that Tangram renders, we will add a more detailed sort rank later

Reply:
Sounds good!